packages/payment-proxy-client/src/App.tsx

```diff
@@ -438,9 +452,10 @@ export default function App() {
                           } as React.CSSProperties
                         }
                       >
-                        Pay & Download
+                        Pay {prettyPrintFIL(selectedFile.size * costPerByte)} &
+                        Download
                       </ProgressButton>
-                    </Box>
+                    </Stack>
```

Comment:
Did this change result in the button being "fully wide"? Be nice to have the button styling consistent with the defaults / the rest of the buttons on the page.